yellowbrick/features/manifold.py

```diff
@@ -236,10 +236,16 @@ def manifold(self, transformer):
         """
         if not is_estimator(transformer):
             if transformer not in self.ALGORITHMS:
-                raise YellowbrickValueError(
-                    "could not create manifold for '%s'".format(str(transformer))
+                    raise YellowbrickValueError("could not create manifold for '{}'".format(str(transformer))
                 )
 
+            if (transformer not in ('tsne','mds','spectral') and self.n_neighbors is None):
+                self.n_neighbors = 10
+                print(
```

Comment:
Rather than doing a `print`, I'd prefer to issue a warning:

```python
import warnings 

from yellowbrick.exceptions import YellowbrickWarning

warnmsg = "using n_neighbors={}; please explicitly pass this argument for the '{}' manifold".format(self.n_neighbors, str(transformer))
warnings.warn(warnmsg, YellowbrickWarning)
```

We will then also need to add a test to make sure this warning is issued.